modules/operations/review.js

```diff
@@ -22,6 +23,17 @@ export function operationReview(selectedIDs, context) {
         return nearestPointOnLine(line, point);
     }
 
+    function getMemberNodes(members) {
+        if (members && members.length > 0) {
+            if (members[0].type === 'relation') {
+                let relationMembers = context.hasEntity(members[0].id).members;
+                return relationMembers[0];
+            } else {
+                return members[0];
+            }
+        }
+    }
```

Comment:
I think there could be situations where the relation has only relation members, so a while loop is in order.
How about something like:
```
    let entity = context.hasEntity(entityId);
    while (entity && entity.type === 'relation') {
        let members = entity.members.map(m => context.hasEntity(m.id));
        let nonRelationMember = members.find(e => e.type !== 'relation');
        if (nonRelationMember) {
            entity = nonRelationMember;
        } else {
            entity = members.find(m => true);
        }
    }

    const entityLoc = getLocation(entity);
```